docs/TemplateManagementCLI.md

```diff
@@ -1,4 +1,4 @@
-Template Management is a tool to manage template files which is used in FHIR dotliquid converter engine. Template manager organizes template files in template image and allows pull and push operations. 
+Template Management CLI is a tool to manage template files for FHIR Converter DotLiquid engine. This CLI tool manages all template files in OCI image format and allows pushing and pulling templates from [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/).
```

Comment:
Might be wrong, from Azure Container Registry => with Azure Container Registry? 😳

Reply:
😳